mod_wasm/modules/wasm/mod_wasm.c

```diff
@@ -260,6 +260,7 @@ static int content_handler(request_rec *r)
     /* creates a new Wasm execution context */
     const char* exec_ctx_id = wasm_executionctx_create_from_config(dcfg->loc);
 
+    int ret = 0;
```

Comment:
This gives off the wrong impression that maybe `ret` is modified in the first `if` and its value might be used for something after the if. I would just use four different `int ret` variables instead.

Reply:
I 100% agree with you but I'm afraid I'm following the convention applied in other modules...